docs/guide/custom-snipptes.md

```diff
@@ -78,6 +83,97 @@ now the page-object contains our new function.:
   }
 ```
 
+* ### CustomSnippets For FillForm
+
+In this type you can extend the [FillForm](./add-meethodes.md#addfillform) methode, to let you fill out your custom form elements.
+The fillForm methode requires a data parameter. You can define the type in the custom snippet.
+
+#### Example
+this custom snippet:
+```ts
+    pageObjectBuilder.customSnippets.addForFillForm({
+        condition: (element) => element.type === 'INPUT', // Add this custom snippet for all elements of type input
+        callBack: async (
+            element: E2eElement,
+            codeBuilder: QueuedCodeBuilder,
+            options: IPageObjectBuilderOptions,
+        ) => {
+            codeBuilder
+                .addLine(`if (data.${Utils.firstCharToLowerCase(element.id)}) {`) // make sure, if the entity on the data object is set
```

Comment:
```suggestion
                .addLine(`if (data.${Utils.firstCharToLowerCase(element.id)}) {`) // make sure, that the property on the data object is set
```

Reply:
if = falls

garantiere, falls die entity auf dem data object gesetzt ist

But it's not important to me